packages/vue/src/examples/pages/error/Error.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div id="error">
+    <SfImage
+      class="image"
+      :src="{
```

Comment:
```suggestion
      :src="require('@storefront-ui/shared/..')
```
for .svg we dont need mobile / desktop files